### PR TITLE
[CU-8685r2bpp] Feature/buzz merchant  my buzz page

### PR DESCRIPTION
**Pull request by lrojas94**
image
image
image

### CODE REVIEW

**Comment by JustMaier** [comment]
Task linked: [CU-8685r2bpp My Buzz Page](https://app.clickup.com/t/8685r2bpp)

**Comment by JustMaier** [comment]
Task linked: [CU-8685r2bpp My Buzz Page](https://app.clickup.com/t/8685r2bpp)

File: src/pages/user/wallet.tsx [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+  },
+  lifetimeBuzz: {
+    animation: `${pulse} 1s ease-in-out infinite`,
+  },
+}));
+
+export default function UserWallet() {
+  const currentUser = useCurrentUser();
+  const theme = useMantineTheme();
+  const { classes, cx } = useStyles();
```
Comment (manuelurenah) [inline src/pages/user/wallet.tsx:137]: Just a little nit
```suggestion
  const { classes, cx, theme } = useStyles();
```

File: src/pages/user/wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+
+  const items: Record<string, number> = useMemo(() => {
+    if (!transactions) return {};
+
+    let start = starterBuzzAmount;
+
+    return transactionsReversed.reduce((acc, transaction) => {
+      const updated = {
+        ...acc,
+        [dayjs(transaction.date).format('DD/MM/YYYY')]: start + transaction.amount,
```
Comment (manuelurenah) [inline src/pages/user/wallet.tsx:160]: Maybe we can use the `formatDate` helper?
```suggestion
        [formatDate(transaction.date, 'DD/MM/YYYY')]: start + transaction.amount,
```

File: src/pages/user/wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+    <Paper withBorder className={classes.featureCard} h="100%">
+      <Stack spacing={4} p="md" align="center" h="100%">
+        <Center>{icon}</Center>
+        <Text weight={500} size="xl">
+          {title}
+        </Text>
+        <Text color="dimmed" align="center">
+          {description}
+        </Text>
+        <Button component="a" mt="auto" w="100%" {...btnProps}></Button>
```
Comment (manuelurenah) [inline src/pages/user/wallet.tsx:553]:
```suggestion
        <Button component="a" mt="auto" w="100%" {...btnProps} />
```

**Review by manuelurenah — APPROVED** [review]

**Pull request merged**